src/components/Modal/Modal.css

```diff
@@ -0,0 +1,95 @@
+
+.modal__wrapper {
+  font-family: Aktiv Grotesk, -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif;
+  position: absolute;
+  z-index: 1000;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+}
+
+.modal__wrapper.--hidden {
+  display: none;
+}
+
+.modal__overlay {
+  background: rgba(0, 0, 0, 0.6);
+  width: 100%;
+  height: 100%;
+  position: fixed;
+}
+
+.modal {
+  position: fixed;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  top: 50%;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  z-index: 2;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  max-width: 43.75rem;
+  max-height: 43.75rem;
+}
+
+@media screen and (max-width: 760px) {
+  .modal {
+    max-height: 100%;
+  }
+}
+
+.modal__content {
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+}
+
+.modal__header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0.6rem 1.2rem;
+}
+
+.modal__left--corner {
```

Comment:
In no parallel dimension can I see `left` as an element, and `corner` as a modifier. I think you meant `.modal__left-corner`? Or maybe `modal__header-left-corner`?